### DISCUSSION TITLE
[Feature] Open image in collection button

### DISCUSSION THREAD

**Discussion by VargaBenceLevente** [immich-app/immich#11109] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If, for example, I go into People > click on any person > a list of images are brought up. However, these lists are incomplete (say I am looking for an image where a particular person turned their back and I know it's on the same day as one that face recognition has found), or I just want to see all photos I took that day.

I believe for this Immich should have functionality similar to [F-Stop Media Gallery](https://www.fstopapp.com/tutorial/)'s "Show in folder" menu. That Android application has albums, smart albums and a search function. So selecting an image > Show in Folder will open the folders view and scroll to the location of the image in that view.

Currently the best way to achieve this is to remember the date, then manually scroll to the date in the full view.

When I started using the application, this is something I immediately tried using instinctively, so I believe that others might, too.

The way I would implement this is by using the Date field in the Details panel. Currently clicking it opens a dialog for changing the date taken of the image. I clicked it expecting to show me other files taken on the same date (as described in this feature request). I would suggest that it should do this instead.

Similarly, I expected the location field to open the map similarly, but it also just opens a dialog to edit the location field, which is also a minor complaint from me.

For platforms, I marked web, but eventually I would request this be done for mobile as well. Should I mark it too?

Also very side sidenote: if you haven't seen F-stop for ideas regarding ideas and (far) future features, I'd recommend taking a look. I have found it very useful in tagging and managing my library, and the tagging feature missing from Immich would be a large pain for me. I am not affiliated, just a satisfied customer.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4115
